edge-currency-bitcoin/src/info/bitcoin.js

```diff
@@ -1,7 +1,7 @@
-// @flow
-import type { EdgeCurrencyInfo } from '../utils/flowTypes.js'
 import type { EngineCurrencyInfo } from '../engine/currencyEngine.js'
 import type { NetworkInfo } from '../utils/bcoinUtils/types.js'
+// @flow
```

Comment:
This is what I was talking about earlier. We need a blank line after the `// @flow` statements so this doesn't happen.

Reply:
fixed